src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/_params.py

```diff
@@ -0,0 +1,72 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from argcomplete.completers import FilesCompleter
+from azure.cli.core.commands import \
+    (register_cli_argument, CliArgumentType)
+from azure.cli.core.commands.parameters import \
+    (tags_type,
+     get_resource_name_completion_list,
+     resource_group_name_type,
+     enum_choice_list)
+
+from azure.cli.command_modules.dla._validators import (validate_resource_group_name,
```

Comment:
This LGTM.
The import of `_params.py` only happens when the command in `dla` module has been found so it wouldn't impact other modules.